src/reducers/index.js

```diff
@@ -259,6 +259,8 @@ function setStateFromProps(state, props) {
   newState.tableSize.useMaxHeight =
     newState.tableSize.height === undefined;
 
+  newState.scrollbarXHeight = props.scrollbarXHeight;
```

Comment:
could you also add these entries to the `getInitialState()` function in reducers/index.js?

Reply:
Should I keep default values as `Scrollbar.SIZE`?

Reply:
took care of this